### ISSUE TITLE
inception_ensemble example uses wrong preprocessor

### ISSUE THREAD

**Issue by philnguyenresson** [triton-inference-server/dali_backend#114]
The [inception_ensemble example](https://github.com/triton-inference-server/dali_backend/blob/main/docs/examples/inception_ensemble/inception_pipeline.py ) uses channel-wise mean/std normalization, whereas I believe the source inception model just rescales to [-1,1] see [here](https://github.com/tensorflow/models/blob/f13be7640980c9c7f65f40d4d7df3fe41a451b2e/research/slim/preprocessing/inception_preprocessing.py#L258)

**Comment by JanuszL** [comment]
Hi @philnguyenresson,

I think there are [many flavors](https://kth.diva-portal.org/smash/get/diva2:1149150/FULLTEXT01.pdf) of the data processing for the inception network. It very much depends on the model you have. For example [Pytorch implementation](https://pytorch.org/hub/pytorch_vision_inception_v3/) uses channel-wise normalization.
Also, this is just an example. You should adjust it to the data processing applied during the training of your models.

**Comment by philnguyenresson** [comment]
Sure, I will make modifications as I need them for my own code. 
I'm just talking about the model given in the example "inception_v3_2016_08_28_frozen", which I believe was trained with [-1,1] normalization
